src/widgets/sticky-header/sticky-header.tsx

```
@@ -0,0 +1,33 @@
+export const StickyHeader = ({
+  isDefault,
+  onToggle,
+  rightSlot,
+}: StickyHeaderProps) => {
+  return (
+    <div className={styles.layout}>
+      <div className={styles.header}>
+        {/** 기본 경험 설정 버튼 */}
+        <button className={styles.button({ isDefault })} onClick={onToggle}>
+          <IconCheck
+            color={isDefault ? themeVars.color.blue600 : themeVars.color.white}
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)